docker-compose.benchmarking.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
```

Comment:
do you want to push this file in this PR ?

Reply:
I'll remove the benchmark service, but Ill leave the mimic service !